### PR TITLE
Add C implementation for `@stdlib/math/base/special/acoversin` 

### LINKED ISSUE #776: [RFC]: Add C implementation for `@stdlib/math/base/special/acoversin`
### Description

This RFC proposes adding C implementation for [`@stdlib/math/base/special/acoversin`](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/acoversin).

```c
#include "stdlib/math/base/special/asin.h"

double stdlib_base_acoversin( const double x ) { 
    return stdlib_base_asin( 1.0 - x );
}
```

### Related Issues

None.

### Questions

No.

### Other

Reference package:

- [`@stdlib/math/base/special/acosh`](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/acosh)

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by aman-095**
Resolves #776.

## Description

> What is the purpose of this pull request?

This pull request adds native C implementation for
`@stdlib/math/base/special/acoversin`

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #776 
-   fixes #

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/math/base/special/acoversin/lib/native.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+*
+* @example
+* var v = acoversin( -1.0 );
+* // returns NaN
+*
+* @example
+* var v = acoversin( NaN );
+* // returns NaN
+*
+*/
```
Comment (Pranavchiku) [inline lib/node_modules/@stdlib/math/base/special/acoversin/lib/native.js]:
```suggestion
* var v = acoversin( NaN );
* // returns NaN
*/
```

File: lib/node_modules/@stdlib/math/base/special/acoversin/manifest.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/special/asin"
+      ]
+    }
+  ]
+}
+
```
Comment (Pranavchiku) [inline lib/node_modules/@stdlib/math/base/special/acoversin/manifest.json]:
```suggestion
}
```

Remove extra end of file line

File: lib/node_modules/@stdlib/math/base/special/acoversin/test/test.native.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+// MODULES //
+
+var resolve = require( 'path' ).resolve;
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var randu = require( '@stdlib/random/base/randu' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var EPS = require( '@stdlib/constants/float64/eps' );
+var tryRequire = require( '@stdlib/utils/try-require' );
```
Comment (Pranavchiku) [inline lib/node_modules/@stdlib/math/base/special/acoversin/test/test.native.js]:
```suggestion
var tryRequire = require( '@stdlib/utils/try-require' );
var EPS = require( '@stdlib/constants/float64/eps' );
var abs = require( '@stdlib/math/base/special/abs' );
```

**Review by Pranavchiku — CHANGES_REQUESTED** [review]
Looks good to me, nice work! Left few comments, apply those and this PR will be good to merge.

File: lib/node_modules/@stdlib/math/base/special/acoversin/test/test.native.js (hunk 3 above)
Comment (Pranavchiku) [reply lib/node_modules/@stdlib/math/base/special/acoversin/test/test.native.js]: This is weird, try this, if it throws linting error you may avoid it.

Comment (aman-095) [reply lib/node_modules/@stdlib/math/base/special/acoversin/test/test.native.js]: It works. Done

**Review by Pranavchiku — APPROVED** [review]
Thanks for this! Looks good to me.

File: lib/node_modules/@stdlib/math/base/special/acoversin/manifest.json [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/special/asin"
+      ]
+    }
+  ]
+}
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/base/special/acoversin/manifest.json:72]: @Planeshifter This is probably another thing we can automate. Ensuring trailing newlines for most file types.

**Pull request merged**